src/plumbing/options/mod.rs

```diff
@@ -236,6 +239,24 @@ pub mod archive {
     }
 }
 
+pub mod branch {
+    #[derive(Debug, clap::Parser)]
+    pub struct Platform {
+        #[clap(subcommand)]
+        pub cmds: Option<Subcommands>,
+
+        /// List remote-tracking as well as local branches.
+        #[clap(long, short = 'a')]
+        pub all: bool,
```

Comment:
This should be with the `List` subcommand.

Reply:
Done.